travis/publish.sh

```diff
@@ -155,11 +155,6 @@ push_docker_master() {
     docker tag "${image}:master" "ghcr.io/${image}:master"
     docker push "ghcr.io/${image}:master"
   done
-
-  echo Building zipkin-builder
-  docker/build_image zipkin-builder latest
```

Comment:
this re-does the build (including downloading stuff from maven), which timed out the last master run